pimd/pim6_cmd.c

```diff
@@ -42,7 +42,94 @@
 #include "pimd/pim6_cmd_clippy.c"
 #endif
 
+DEFPY (ipv6_pim_rp,
+       ipv6_pim_rp_cmd,
+       "ipv6 pim rp X:X::X:X$rp [X:X::X:X/M]$group",
+       IPV6_STR
+       PIM_STR
+       "Rendevous Point\n"
+       "ipv6 address of RP\n"
+       "Group Address range to cover\n")
+{
+	const char *vrfname;
+	char rp_group_xpath[XPATH_MAXLEN];
+
+	vrfname = pim_cli_get_vrf_name(vty);
+	if (vrfname == NULL)
+		return CMD_WARNING_CONFIG_FAILED;
+
+	snprintf(rp_group_xpath, sizeof(rp_group_xpath),
+		 FRR_PIM_STATIC_RP_XPATH,
+		 "frr-pim:pimd", "pim", vrfname, "frr-routing:ipv6",
+		 rp_str);
+	strlcat(rp_group_xpath, "/group-list", sizeof(rp_group_xpath));
+
+	if (!group_str)
+		nb_cli_enqueue_change(vty, rp_group_xpath, NB_OP_CREATE,
+				"FF00::0/8");
+	else
+		nb_cli_enqueue_change(vty, rp_group_xpath, NB_OP_CREATE,
+				group_str);
+
+	return nb_cli_apply_changes(vty, NULL);
+}
+
+DEFPY (no_ipv6_pim_rp,
+       no_ipv6_pim_rp_cmd,
+       "no ipv6 pim rp X:X::X:X$rp [X:X::X:X/M]$group",
+       NO_STR
+       IPV6_STR
+       PIM_STR
+       "Rendevous Point\n"
```

Comment:
Rendezvous Point ?

Reply:
Thanks @rgirada for pointing it out. Have fixed it.